specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -808,6 +808,59 @@
         }
       }
     },
+    "/keys/{key-name}/{key-version}/export": {
+      "post": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "exportKey",
+        "summary": "Exports a key.",
+        "description": "The export key operation is applicable to all key types. The target key must be marked exportable. This operation requires the keys/export permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key to get."
+          },
+          {
+            "name": "key-version",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Adding the version parameter retrieves a specific version of a key."
```

Comment:
Seems copied and pasted, and really doesn't fit here. If it's required, they have to specify it.

Reply:
if you mean key-version, then it is marked as required in GetKey as well and it is not actually required in the protocol there either?

Reply:
Interesting. That key-version is required for GetKey is wrong. We actually treat it optionally in the SDK (cross-lang) and it works. I'm just saying that here, "Adding the version..." makes it sound like it's optional. If it's not, I would suggest being as assertive as for key-name.

Reply:
So it's true that version is not required in many places in the swagger where it is documented as required. I think this is because historically the code generation did not support non-required path parameters. If this is no longer the case perhaps we should document this newer one properly and fix the documentation on others in a separate PR.